docusaurus/docs/iOS/guides/working-with-the-composer.md

```diff
@@ -0,0 +1,221 @@
+```swift
+final class CustomComposerVC: ComposerVC {
+    override func setUp() {
+        super.setUp()
+
+        composerView.emojiButton.addTarget(self, action: #selector(showEmojiPicker), for: .touchUpInside)
+    }
+
+    @objc open func showEmojiPicker(sender: UIButton) {
+        // Custom implementation of opening the emoji picker
+    }
+}
+
+components.messageComposerVC = CustomComposerVC.self
+```
+
+Besides adding new actions, it is also possible to customize the existing actions, for example, an application might want to present the attachments picker on a completely different view and with different logic:
+```swift
+final class CustomComposerVC: ComposerVC {
+    override func showAttachmentsPicker(sender: UIButton) {
+        // Custom implementation of opening the attachments picker
+    }
+}
+
+components.messageComposerVC = CustomComposerVC.self
+```
+
+## Customizing Typing Suggestions
+When typing on the composer, if you type `@` it will present a view suggesting users that you can mention, or if you type `/` as the first character, it will show command suggestions. You can customize not only the view of the suggestions but also the logic around it.
+
+### Changing the Recognized Symbol
+If your app has a different character to recognize user mentions or commands, then you just need to override the symbols like this:
+```swift
+final class CustomComposerVC: ComposerVC {
+    override var mentionSymbol: String { "👤" }
+
+    override var commandSymbol: String { "⚡️" }
+}
+
+components.messageComposerVC = CustomComposerVC.self
+```
+That's it, with this in place, the suggestions will now be triggered when you type your custom symbol.
+
+### Customizing the Suggestions UI
+The UI SDK provides a `ChatSuggestionsVC` component responsible for showing user suggestions and commands suggestions.
+
+#### Basic Customization
+In case you only want to change the styling of the container that shows both suggestions, you can override the `ChatSuggestionsVC`. Here is an example of a basic customization of the container component:
+```swift
+final class CustomChatSuggestionsVC: ChatSuggestionsVC {
+    override func setUpAppearance() {
+        super.setUpAppearance()
+
+        collectionView.layer.borderWidth = 1
+        collectionView.layer.borderColor = appearance.colorPalette.border.cgColor
+        collectionView.layer.cornerRadius = 10
+        view.layer.shadowOpacity = 0.0
+    }
+}
+
+components.suggestionsVC = CustomChatSuggestionsVC.self
+```
+Here we simply remove the shadow and customize the border style of the suggestions component.
+
+**Result:**
+
+| Before  | After |
+| ------------- | ------------- |
+| <img src={require("../assets/composer-suggestionsVC-before.png").default} width="100%"/> | <img src={require("../assets/composer-suggestionsVC-after.png").default} width="100%"/> |
```

Comment:
TBH it looks like the first image is blurry and second is more clear... we need a better example for this, like sharp borders or getting rid of borders